ingest/config/annotations.tsv

```diff
@@ -234,4 +234,9 @@ MN185328	country	French Guiana
 MN185328	division	French Guiana
 MN185328	location	French Guiana
 MN185328	region	South America
-KY328289	date	2016-05-15
\ No newline at end of file
+KY328289	date	2016-05-15
+MW165884	country	USVI
```

Comment:
_not a request for changes_

I was expecting a change in the [(local) geolocation rules](https://github.com/nextstrain/zika/blob/main/ingest/config/geolocation-rules.tsv), but this works too!

Reply:
That was my expectation too, since this is overwriting the [geolocation rules from ncov-ingest](https://github.com/nextstrain/ncov-ingest/blob/d3fa3b990ea417e3188537e1125f032169e55b69/source-data/gisaid_geoLocationRules.tsv), which include
```
$ grep -i 'virgin islands' source-data/gisaid_geoLocationRules.tsv 
*/Virgin Islands/St. Thomas/	North America/USA/Virgin Islands/St. Thomas
North America/USA/Vi/*	North America/USA/Virgin Islands/*
North America/USA/Virgin Islands of the U.S./*	North America/USA/Virgin Islands/*
North America/USA/U.S. Virgin Islands/*	North America/USA/Virgin Islands/*
North America/USA/Us Virgin Islands/*	North America/USA/Virgin Islands/*
North America/USA/Virgin Islands Of The U.S./*	North America/USA/Virgin Islands/*
North America/Caribbean/British Virgin Islands/	Europe/United Kingdom/British Virgin Islands/
North America/British Virgin Islands/*/*	Europe/United Kingdom/British Virgin Islands/*
North America/U.S. Virgin Islands//	North America/USA/Virgin Islands/
North America/U.S. Virgin Islands/U.S. Virgin Islands/	North America/USA/Virgin Islands/
North America/U.S. Virgin Islands/St. Thomas/	North America/USA/Virgin Islands/St. Thomas
North America/U.S. Virgin Islands/St. Croix/	North America/USA/Virgin Islands/St. Croix
South America/British Virgin Islands//	Europe/United Kingdom/British Virgin Islands/
South America/British Virgin Islands/British Virgin Islands/	Europe/United Kingdom/British Virgin Islands/
South America/British Virgin Islands/Caribbean/	Europe/United Kingdom/British Virgin Islands/
```

If we want to special case zika to use USVI, I _think_ this can be done with the local geolocation rule:
```
North America/USA/Virgin Islands/*	North America/USVI/USVI/*
```

Reply:
Is my understanding correct that we want to label all "Virgin Islands" records as "USVI" not just the ones associated with this https://github.com/blab/zika-usvi/ study?

If so, is there a reason the zika build should use "USVI" while ncov, mpox, and other pathogens use "Virgin Islands"? I'm happy to add it here, just thinking through if this is a good design decision.

Alternatively, I'm happy to reverse this change and use "Virgin Islands" instead, as suggested by the [2nd solution here](https://github.com/nextstrain/zika/issues/40#issue-2155030545) as being the better solution going forward.

> 2. Change the spiked-in sequences to "Virgin Islands" to match GenBank.

Reply:
Yes, that's my understanding based upon the language used in the [paper](https://www.biorxiv.org/content/10.1101/113100v1.full.pdf+html) and the country strings we were using while maintaining the Zika tree during the outbreak.

Reply:
Thanks all! Switched to the local geolocation rule, and tried to summarize the rational in the commit message: https://github.com/nextstrain/zika/pull/41/commits/927222f130b83a0bdcadea7892f6e811a61d9fa6 

Please let me know if you catch anything else